hw18.py

```diff
@@ -0,0 +1,25 @@
+class Bot:
+    def __init__(self, name, message = ""):
+        self.name = name
+    def say_name(self):
+        print(self.name)
+    def send_message(self, message):
+        print(self.message)

```

Comment:
you need to print here `message` that you get during the function call, not `self.message`